sematic/metrics/func_success_rate_metric.py

```diff
@@ -0,0 +1,78 @@
+# Standard Library
+from datetime import datetime
+from typing import Optional, Tuple
+
+# Third-party
+from sqlalchemy.orm import Query, Session, joinedload
+
+# Sematic
+from sematic.abstract_future import FutureState
+from sematic.abstract_system_metric import AbstractSystemMetric
+from sematic.db.models.run import Run
+from sematic.metrics.metric_point import MetricType
+
+FINAL_STATES = {FutureState.RESOLVED, FutureState.FAILED, FutureState.NESTED_FAILED}
+
+
+class FuncSuccessRateMetric(AbstractSystemMetric):
+    """
+    System Metric to measure Sematic Function's success rate.
+
+    Success rate is defined as:
+
+    Number of qualifying RESOLVED runs
```

Comment:
nit: it might be more clear that this is a fraction if either:
1 center the `Number of qualifying RESOLVED runs`
or 2. Express explicitly
Numerator: Number of qualifying RESOLVED runs
Denominator: Number of qualifying (RESOLVED + FAILED + NESTED_FAILED) runs. 
(The instruction doc for US Tax From 1040 uses the latter form)